ChromecastSampleApp/ChromecastSampleApp/Players/PlayerViewController.m

```diff
@@ -87,7 +87,9 @@ - (void)viewDidLoad {
   // Init the castManager in the ooyalaPlayer
   [self.ooyalaPlayer initCastManager:self.castManager];
   [self.ooyalaPlayer setEmbedCode:self.embedCode];
-  [self.ooyalaPlayer play];
+  if( self.mediaInfo.startPlaying ) {
```

Comment:
fair enough. it could be "If CastPlayer is casting, don't bother autoplaying." that does make more sense